openmc/waste.py

```diff
@@ -0,0 +1,279 @@
+            return "Class A"
+        elif ratio1 < 1.0:
+            return "Class C"
+        else:
+            return "GTCC"
+
+    elif table2_nuclides_present:
+        # Classification based on §61.55(a)(4)
+        return classify_table2(*ratio2)
+
+    else:
+        # Classification based on §61.55(a)(6)
+        return "Class A"
+
+
+def _waste_disposal_rating(
+        mat: openmc.Material,
+        limits: str | dict[str, float] = 'Fetter',
+        metal: bool = False,
+    ) -> float:
+    """Return the waste disposal rating for a material.
+
+    This method returns a waste disposal rating for the material based on a set
+    of specific activity limits. The waste disposal rating is a single number
+    that represents the sum of the ratios of the specific activity for each
+    radionuclide in the material against a nuclide-specific limit. A value less
+    than 1.0 indicates that the material "meets" the limits whereas a value
+    greater than 1.0 exceeds the limits.
+
+    Parameters
+    ----------
+    mat : openmc.Material
+        The material to classify.
+    limits : str or dict, optional
+        The name of a predefined set of specific activity limits or a dictionary
+        that contains specific activity limits for radionuclides, where keys are
+        nuclide names and values are activities in units of [Ci/m3]. The
+        predefined options are:
+
+        - 'Fetter': Uses limits from Fetter et al. (1990)
+        - 'NRC_long': Uses the 10 CFR 61.55 limits for long-lived radionuclides
+        - 'NRC_short_A': Uses the 10 CFR 61.55 class A limits for short-lived
+          radionuclides
+        - 'NRC_short_B': Uses the 10 CFR 61.55 class B limits for short-lived
+          radionuclides
+        - 'NRC_short_C': Uses the 10 CFR 61.55 class C limits for short-lived
+          radionuclides
+    metal : bool, optional
+        Whether or not the material is in metal form (only applicable for NRC
+        based limits)
+
+    Returns
+    -------
+    float
+        The waste disposal rating for the material.
+
+    """
+    if limits == 'Fetter':
+        # Specific activity limits for radionuclides with half-lives between 5
+        # years and 1e12 years from Table 2 in Fetter
+        limits = {
+            "Be10": 5.0e3,
+            "C14": 6.0e2,
+            "Al26": 9.0e-2,
+            "Si32": 6.0e2,
+            "Cl36": 1.0e1,
+            "Ar39": 2.0e4,
+            "Ar42": 2.0e4,
+            "K40": 2.0e0,
+            "Ca41": 1.0e4,
+            "Ti44": 2.0e2,
+            "Fe60": 1.0e-1,
+            "Co60": 3.0e8,
+            "Ni59": 9.0e2,
+            "Ni63": 7.0e5,
+            "Se79": 5.0e1,
+            "Kr81": 3.0e1,
+            "Sr90": 8.0e5,
+            "Nb91": 2.0e2,
+            "Nb92": 2.0e-1,
+            "Nb94": 2.0e-1,
+            "Mo93": 4.0e3,
+            "Tc97": 4.0e-1,
+            "Tc98": 1.0e-2,
+            "Tc99": 6.0e-2,
+            "Pb107": 9.0e2,
```

Comment:
```suggestion
            "Pd107": 9.0e2,
```
Should be palladium not lead

Reply:
:man_facepalming: I was hoping that someone would check the numbers since I had to manually put these all in. Thanks a ton!